config/routes.rb

```diff
@@ -61,4 +61,6 @@
       post :stop_impersonating, on: :collection
     end
   end
+
+  mount ActionCable.server => '/cable'
```

Comment:
later on production do this https://www.phusionpassenger.com/library/config/nginx/action_cable_integration/